packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -880,7 +894,9 @@ export class ZWaveController extends EventEmitter {
 					this._nodes.set(newNode.id, newNode);
 					this._nodePendingInclusion = undefined;
 
-					await this.secureBootstrapS0(newNode);
+					if (!this._includeNonSecure) {
```

Comment:
@AlCalzone Wan't it better to add this check inside the function?

Reply:
In this way where do you set secure to false?

Reply:
It felt wrong to call a function that does secure bootstrapping if you know you don't want it.
The security CC is removed when one of the commands fails (in the method)